ios/brave-ios/App/l10n/tools/push-strings-to-crowdin.sh

```diff
@@ -0,0 +1,171 @@
+fi
+
+cd $(dirname "$0")
+
+echo "Exporting strings from Xcode project..."
+(cd ../../ && xcodebuild -exportLocalizations SWIFT_EMIT_LOC_STRINGS=NO) >>output.log 2>&1
+if [ $? != 0 ] ; then
+  report_error 4 "ERROR: Failed to export strings from Xcode project, please see output.log"
+fi
+
+mv -f ../../Client/en.xcloc/Localized\ Contents/en.xliff . >>output.log 2>&1
+
+if [ $? != 0 ] ; then
+  report_error 3 "ERROR: Failed to export strings from Xcode project, please see output.log"
+fi
+
+echo "Cleaning up strings..."
+./xliff-cleanup.py en.xliff
+if [ $? != 0 ] ; then
+  report_error 4 "ERROR: Failed to cleanup strings, please see output.log"
+fi
+
+sed -i '' 's/Shared\/Supporting Files/brave/' en.xliff >>output.log 2>&1
+if [ $? != 0 ] ; then
+  report_error 4 "ERROR: Failed to cleanup strings, please see output.log"
+fi
+
+crowdin_project_id=33
+crowdin_storage_id=null
+
+add_storage_id()
+{
+  FILE_PATH="en.xliff"
+  result=$(curl --silent \
+    -X "POST" "https://brave-software.crowdin.com/api/v2/storages" \
+    -H "Authorization: Bearer ${TOKEN}" \
+    -H 'Content-Type: application/octet-stream' \
+    -H "Crowdin-API-FileName: en.xliff" \
+    --data-binary @"$FILE_PATH"
+  )
+  storage_id=$(echo "$result" | jq '.data.id')
+  echo $storage_id
+}
+
+delete_storage_id()
+{
+  http_status_code=$(curl --silent -o /dev/null -w "%{http_code}" \
+  -X "DELETE" "https://brave-software.crowdin.com/api/v2/storages/$1" \
+  -H "Authorization: Bearer ${TOKEN}" \
+  -H "Content-Type: application/json"
+  )
+  echo $http_status_code
+}
+
+add_file()
+{
+  http_status_code=$(curl --silent -o /dev/null -w "%{http_code}" \
+    -X "POST" "https://brave-software.crowdin.com/api/v2/projects/$crowdin_project_id/files" \
+    -H "Authorization: Bearer ${TOKEN}" \
+    -H 'Content-Type: application/json' \
+    --data @- << EOF
+      {
+        "storageId": $crowdin_storage_id,
+        "name": "en.xliff",
+        "type": "xliff"
+      }
+EOF
+  )
+  echo $http_status_code
+}
+
+delete_file()
+{
+  http_status_code=$(curl --silent -o /dev/null -w "%{http_code}" \
+    -X "DELETE" "https://brave-software.crowdin.com/api/v2/projects/$crowdin_project_id/files/$1" \
+    -H "Authorization: Bearer ${TOKEN}" \
+    -H 'Content-Type: application/json' \
+  )
+  echo $http_status_code
+}
+
+echo "Creating a storage id in Crowdin..."
+crowdin_storage_id=$(add_storage_id)
+if [ $crowdin_storage_id == null ]
+then
+  report_error 5 "ERROR: Failed to create a storage id in Crowdin"
+else
+  echo "Crowdin Storage ID: $crowdin_storage_id"
+fi
+
+echo "Checking if there is already a file id for our project in Crowdin..."
+file_result=$(curl --silent \
+  -X "GET" "https://brave-software.crowdin.com/api/v2/projects/$crowdin_project_id/files" \
+  -H "Authorization: Bearer ${TOKEN}" \
+  -H "Content-Type: application/json"
+)
+file_id=$(echo "$file_result" | jq '.data.[0].data.id')
+if [ $file_id == null ]
```

Comment:
Is this checking for the string literal "null"?

Reply:
updated in [703406a](https://github.com/brave/brave-core/pull/27230/commits/703406ad7301f0b75556ab25732527ffcc0b75be)